crates/db_schema/src/source/local_site_url_blocklist.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(feature = "full")]
+use crate::schema::local_site_url_blocklist;
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+#[cfg(feature = "full")]
+use ts_rs::TS;
+
+#[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
```

Comment:
This is missing `skip_serializing_none` like the other structs, it needs to be added otherwise lemmy-js-client will have some issues.

Reply:
Ah, my bad.

Reply:
np